index.js

```diff
@@ -0,0 +1,16 @@
+"use strict"
+
+module.exports = {
+  extends: [
+    'airbnb-base',
+    'airbnb-typescript/base',
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking"
+  ],
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
```

Comment:
In [keep-network/eslint-config-keep](https://github.com/keep-network/eslint-config-keep/blob/0c27ade54e725f980e971c3d91ea88bab76b2330/index.js#L19-L22) we use `prettier`, do we also want to have it here?

Reply:
I see it's growing :) the intent was to lint typescript only, but since we're heading towards two repos `@thesis-co/eslint-config` and `@thesis-co/eslint-config-typescript` then I can add `prettier` to `@thesis-co/eslint-config`